pocs/focuser/birger.py

```diff
@@ -56,7 +56,7 @@ def __init__(self,
                  name='Birger Focuser',
                  model='Canon EF-232',
                  initial_position=None,
-                 dev_node_pattern='/dev/tty.USA49WG*.?',
+                 dev_node_pattern='/dev/tty.USA49*.?',
```

Comment:
This is ultimately dependent on those acroname usb hubs, so we should make an appropriate comment on what this means.

Reply:
Yes, should have better documentation.  This is specific to the Tripplite Keyway USA-49 USB-serial adaptors, rather than the usb hubs. At least I made it an argument and only hard coded the default value...

Reply:
Docstring written.